Add events support group charter first draft (#46)
django/dsf-working-groups: active/events-support.md

# Events Support Working Group

## Scope of responsibilities

The group supports organizers of Django-related events globally, including DjangoCon conferences. The group acts on behalf of the DSF Board to:

- Be the point of contact for organizers to ask questions and seek guidance.
- Proactively provide support to event organizers.
- Debrief with organizers after events.
- Maintain shared resources to facilitate continuity and knowledge transfer between events.
- Actively promote diversity and inclusion in event support practices, encouraging organizers to create inclusive events that welcome and represent diverse communities.

For DjangoCon events, the working group will:

- Liaise with DEFNA (Django Events Foundation North America) who oversee DjangoCon US.
- Manage and execute the Call for Organizers process for DjangoCon Europe, providing recommendations to the DSF board for final approval.
- Assist DjangoCon Europe organizers with event planning, logistics, and execution.

> - Work with DjangoCon Africa organizers to make this event sustainable.

Reviewer: ```suggestion
- Work with DjangoCon Africa organizers to make this event sustainable.
- Work with the Social Media Working Group to promote events
```
Would it make sense?

Author: I think so! Not sure in what way, where the remit of the social media group is and what’s for this events group.

# Events Support Working Group

## Scope of responsibilities

The group supports organizers of Django-related events globally, including DjangoCon conferences. The group acts on behalf of the DSF Board to:

- Be the point of contact for organizers to ask questions and seek guidance.
- Proactively provide support to event organizers.
- Debrief with organizers after events.
- Maintain shared resources to facilitate continuity and knowledge transfer between events.
- Actively promote diversity and inclusion in event support practices, encouraging organizers to create inclusive events that welcome and represent diverse communities.

For DjangoCon events, the working group will:

- Liaise with DEFNA (Django Events Foundation North America) who oversee DjangoCon US.

> - Manage and execute the Call for Organizers process for DjangoCon Europe, providing recommendations to the DSF board for final approval.

Reviewer 2: I think practically speaking, reviewing DCEU applications is about having individual group members add comments in documents. We can't ever have a process with a real consensus in the team before the board is briefed?

Well, maybe we could and should?

I'd like to add something about having consensus at least about some kind of general criteria that applications are processed through...

```suggestion
- Manage and execute the Call for Organizers process for DjangoCon Europe, providing the DSF board with both a non-binding list of general criteria AND reviews of concrete proposals.
```

Trying to not split this into several items, but my main idea here is:

As a member of the previous group, I found it harder to share preferences once specific applications had come in (IIRC, I barely added any comments).

That's why, I think instead, maybe the group can discuss a set of general guidelines for applications before any proposals are processed. That will also make it easier for the group member to review an application.

Examples of criteria I have in mind are: inclusion vs. economic exclusion, transportation issues, and governance of profits.

Secondly, that criteria will either feed into the call itself, or the vetting process, i.e. giving feedback to applications BEFORE they are finally submitted for a board review (but that's also the process that's been a bid muddy.. is it the board or the group that helps out in this process?)

Reviewer 2: Btw this comment is intentionally not part of my approving review - I don't think we need to block anything.

- Assist DjangoCon Europe organizers with event planning, logistics, and execution.
- Work with DjangoCon Africa organizers to make this event sustainable.
- Work with the Social Media Working Group to promote events

## Initial membership

The group has a small initial membership, with the goal of expanding to 7-10 members able to represent

Current members:

- Chair: Thibaud Colas
- Co-Chair: David Vaz
- Board Liaison: Thibaud Colas
- Other members:

>   - You?

Reviewer 2: ```suggestion
  . Benjamin Balder Bach
  - You?
```

Reviewer 2: I'm nominating myself with two skills:

* Being a member of the former DjangoCon Europe Support Group
* Being an organizer of DjangoCon Europe 2019

I'd like to continue to help foster happy, creative, sustainable Django-related events. Both big, but especially also small :)
